kss-config.json

```diff
@@ -2,7 +2,8 @@
   "source": "scss",
   "destination": "styleguide",
   "css": [
-    "css/decanter.css"
+    "css/decanter.css",
```

Comment:
Do we need both files here or can this be consolidated in to one?

Reply:
I believe we need both, because they are being compiled separately in order to keep the kss-specific stylesheet out of Decanter itself.